.gitignore

```diff
@@ -1,2 +1,4 @@
 .syncconfig
 .expanded/
+expand2høyre.sh
```

Comment:
I think you should move this into .høyre, so you could just add a wildcard for .* (weird to have Høyre hardcoded into the connector)

Reply:
Fair point. I'll fix.

Reply:
Fixed.